kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/KafkaCommandDecoder.java

```diff
@@ -257,6 +257,7 @@ public void channelRead(ChannelHandlerContext ctx, Object msg) throws Exception
             if (!isActive.get()) {
                 handleInactive(kafkaHeaderAndRequest, responseFuture);
             } else {
+                log.info("process {}", kafkaHeaderAndRequest.getHeader().apiKey());
```

Comment:
We should remove this line. Otherwise there will be a line of logs each time a request arrived.

Reply:
good catch ! it was a debug